packages/passport-auth/lib/Passport.js

```diff
@@ -89,59 +94,86 @@ class PassportAuthStrategy {
       validationArgs
     );
 
-    // Lookup a past, verified session, that links to a user
-    let pastSessionItem;
-    let fieldItemPopulated;
-    try {
-      // NOTE: We don't need to filter on verifiedAt as these rows can only
-      // possibly exist after we've validated with Passport Service (see above)
-      pastSessionItem = await this.getSessionList().adapter.findOne({
-        [FIELD_SERVICE_NAME]: this.authType,
-        [FIELD_USER_ID]: validatedInfo.id,
-      });
-      // find user item related to past session, join not possible atm
-      fieldItemPopulated =
-        pastSessionItem &&
-        (await this.getList().adapter.findById(pastSessionItem[this.config.itemField].toString()));
-    } catch (sessionFindError) {
-      // TODO: Better error message. Why would this fail? DB connection lost? A
-      // "not found" shouldn't throw (it'll just return null).
-      throw new Error(`Unable to lookup existing ${this.authType} sessions: ${sessionFindError}`);
-    }
-
     const newSessionData = {
       [this.config.tokenSecretField]: accessToken,
       [FIELD_SERVICE_NAME]: this.authType,
       [FIELD_USER_ID]: validatedInfo.id,
       [FIELD_USERNAME]: validatedInfo.username,
     };
 
-    // Only add a reference to the parent list when we know the link exists
-    if (pastSessionItem) {
-      newSessionData[this.config.itemField] = fieldItemPopulated.id;
+    let itemId;
+    // Find an existing user that matches the given validated service id
+    try {
+      const queryName = this.getList().gqlNames.listQueryName;
+      const data = await request(
```

Comment:
I've added an issue to come back and fix this later: https://github.com/keystonejs/keystone-5/issues/1300